inst/ms/aggreCAT.qmd

```diff
@@ -450,20 +477,22 @@ transparency of reporting, and journal reporting [@Hanea2021]. Within
 distributed as columns in the dataset, and for each claim `paper_id`,
```

Comment:
L472: `data_supp_reasons` wont have all 25 codes, just those that passed ICR. `data_supp_reasons_full` will have all 25, but that might not be in the package (and doesn't really need to be)

Reply:
And should note that all reason code columns have to start with `RW`